packages/devcmd/src/process/ProcessExecutor.ts

```diff
@@ -13,13 +13,17 @@ export interface ProcessInfo {
   };
 }
 
+/**
+ * An interface that describes the behavior of a target,
+ * that can display log and error messages. 
+ */
 export interface ConsoleLike {
   log(message?: any, ...optionalParams: any[]): void;
   error(message?: any, ...optionalParams: any[]): void;
 }
 
 class SafeConsoleLike implements ConsoleLike {
-  constructor(private readonly consoleLike: ConsoleLike | undefined | null) {}
+  constructor(private readonly consoleLike: ConsoleLike | undefined | null) { }
```

Comment:
hmmm, when I opened and saved this file on my machine, the space between the curlies here, and some other lines (e.g. trailing spaces) were removed.

please ensure that you have prettier and the prettier plugin for VS code installed, and that auto-format-on-save works for you (it should be enabled in the workspace settings), so that we can keep the styling consistent here

Reply:
update: this now also failed the lint-check in our GH pipeline 🙊 
I've approved your changes to run in our pipeline, so I think further updates should automatically get checked there as well :)

Reply:
I only had Prettier installed in my WSL2 instance of VSCode, but not within the development container from #26 ^^